Dockerfile

```diff
@@ -17,4 +19,25 @@ COPY  poetry.lock pyproject.toml /project/
 COPY docker-flask-entrypoint.sh ./docker-entrypoint.sh
 RUN cd /project/
 RUN poetry install
-CMD ["./docker-entrypoint.sh"]
\ No newline at end of file
+CMD ["./docker-entrypoint.sh"]
+
+FROM base as test
+RUN apt-get update && apt-get install curl fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libatspi2.0-0 libcairo2 libcups2 libdbus-1-3 libdrm2 libgbm1 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libx11-6 libxcb1 libxcomposite1 libxdamage1 libxshmfence1 wget xdg-utils -y 
+RUN apt-get install fonts-liberation libasound2 
+COPY /tests/ /project/tests/
+COPY /tests_e2e/ /project/tests_e2e/
+COPY  poetry.lock pyproject.toml /project/
+COPY docker-flask-entrypoint.sh ./docker-entrypoint.sh
+RUN cd /project/
+RUN poetry install
+RUN curl -sSL https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o chrome.deb &&\
```

Comment:
Not super important, but if you intend to run these tests often you may find it painful waiting for this install to re-run every time you change your test files. Moving these steps before the COPY layers would make sense